pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -118,13 +118,16 @@ def __call__(self, time: float):
                     max_weight = np.nanmax(distance_weights)
                     distance_weights = np.nan_to_num(distance_weights,
                                                      nan=max_weight)
+                # Use of the cutoff distance idea from CovidSim.
+                cutoff = Parameters.instance().infection_radius
+                distance_weights = [weight if (weight > 1/cutoff) else 0 for
+                                    weight in distance_weights]
```

Comment:
I though you had added checks that this list is of the right length (and has non-zero elements) but can't see them here?

Reply:
Do now :)